### PR TITLE
Use Yarn instead of NPM

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Other

### CODE REVIEW

**Review by gabrieldutra — APPROVED** [review]
Note that these changes seemed to have increased CircleCI time in 10 minutes (maybe due to yarn install?)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
-FROM node:12 as frontend-builder
+FROM node:14.17 as frontend-builder
+
+RUN npm install --global --force yarn@1.22.10
```
[the remark this answers is not in the document]
Comment (kravets-levko) [reply Dockerfile:3]: They do, but with older version (different in both cases). For consistency, I'm upgrading it everywhere

**Comment by kravets-levko** [comment]
> Note that these changes seemed to have increased CircleCI time in 10 minutes (maybe due to yarn install?)

Thank you for pointing, I'll double-check

**Comment by kravets-levko** [comment]
So I checked it, and looks most of the extra time (4-7 minutes in total) comes from Yarn upgrade. For now I'll keep it as is - as there's more migration work coming. After it's fully done - I'll revisit all performance issues (will create a ticket for this)

**Comment by ghost** [comment]
Super Dikess

**Pull request merged**